html-forms/index.html

```diff
@@ -19,17 +19,19 @@
     </select>
 
 
-    <label for="fav-lang" >Favourite language (choose one)</label>
-    <input type="radio" name="language" id= "fav-lang"value="HTML" check="checked"/> HTML
-    <input type="radio" name="language"  id="fav-lang" value="css" check="checked" /> CSS
-    <input type="radio" name="language"    id="fav-lang" value="js" check="checked" /> Javascript
+
+    <label for="fav-lang">Favourite language (choose one)
+      <input type="radio" name="language" id="html-lang" value="HTML" check="checked" /> HTML
+      <input type="radio" name="language" id="css-lang" value="css" check="checked" /> CSS
+      <input type="radio" name="language" id="js-lang" value="js" check="checked" /> Javascript
+    </label>
```

Comment:
You should have a `label` element for each of the inputs here, not all of them inside one `label`